amgcl/coarsening/plain_aggregates.hpp

```diff
@@ -61,132 +61,134 @@ namespace coarsening {
  * \ingroup aggregates
  */
 struct plain_aggregates {
-    /// Aggregation parameters.
```

Comment:
Can you please use spaces for indentation here to keep the diff minimal (and to keep the file consistent with the rest of the code)?